### ISSUE TITLE
e2e-attributes are translated causing tests to fail

### ISSUE THREAD

**Issue by theobadha** [ushahidi/platform-client-mzima#695]
Some english values to unique attributes are apparently getting translated too in the CI environment. This is causing associated tests to fail since the tests look for the attributes in plain english. 
Correct this so that the attributes are not translated
This is happening for the categories page. 

<img width="1488" alt="Screenshot 2023-11-07 at 10 35 34" src="https://github.com/ushahidi/platform-client-mzima/assets/38259840/737d0db7-9ded-4e47-9d1b-b5430ac705c4">

**Comment by oyenuga17** [comment]
@AmTryingMyBest please I'd love to look into this

**Comment by theobadha** [comment]
@oyenuga17  Please hold on, there's another PR in draft that may have a fix for this, I'll need to check that out first.
